src/core/bridge.ts

```diff
@@ -68,11 +68,12 @@ export abstract class AbstractBridge {
   }
 
   private extractRequest(query: any): { request: Request; queryParams: any; } {
+    const queryParams = Object.assign({}, this.config['customUrlParams'], query.queryParams);
```

Comment:
also, we can only get `query.queryParams` in the case where type is `object` and `query instanceof Query`